app/views/profiles/_status.html.slim

```diff
@@ -0,0 +1,16 @@
+.float-right
```

Comment:
I would prefer this to be a `Block` in BEM terminology. That way all the classes below can be scoped  to this block, and there's no need to add `.status-label` to the global CSS namespace.